packages/compartment-mapper/src/import-hook.js

```diff
@@ -18,13 +18,15 @@ const resolveLocation = (rel, abs) => new URL(rel, abs).toString();
  * @param {string} baseLocation
  * @param {Sources} sources
  * @param {Record<string, CompartmentDescriptor>} compartments
+ * @param {Record<string, any>} exitModules
```

Comment:
I may be wrong, but please see if the following hunch expresses what you mean:
```suggestion
 * @param {Record<string, never>} exitModules
```

Reply:
This is perfect. Thank you.